### PR TITLE
Dsp 845

### PR DESCRIPTION
**Pull request by Najon**
Hi,

in order to get hexagon sdk working on db845c these change needs to be pulled.

Let me know if there is any issue.

Best regards,
Julien

### CODE REVIEW

File: recipes-support/adsprpc/adsprpc_git.bb~ [hunk 1]
```diff
@@ -0,0 +1,15 @@
+SUMMARY = "Qualcomm ADSPRPC applications and library"
```
Comment (ndechesne) [inline recipes-support/adsprpc/adsprpc_git.bb~]: please delete this file.

File: recipes-support/adsprpc/adsprpc_git.bb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e868d7526728e9b1c0c4f34730d27754"
+
+SRCREV = "741c1415df731503b3aea8b754223a871081d2f3"
+SRC_URI = "git://git.linaro.org/landing-teams/working/qualcomm/libadsprpc.git;branch=master;protocol=https"
+
+PV = "0.0+${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+inherit autotools
```
Comment (ndechesne) [inline recipes-support/adsprpc/adsprpc_git.bb]: missing newline at end of file.

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb [hunk 3]
```diff
@@ -12,18 +12,21 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
...
 do_install() {
     install -d ${D}${nonarch_base_libdir}/firmware/
     install -d ${D}${nonarch_base_libdir}/firmware/qcom/venus-5.2/
+    install -d ${D}${nonarch_base_libdir}/firmware/qcom/db845c
 
     install -m 0444 ./17-USB3-201-202-FW/K2026090.mem ${D}${nonarch_base_libdir}/firmware/
     install -m 0444 ./18-adreno-fw/a630*.* ${D}${nonarch_base_libdir}/firmware/
     install -m 0444 ./20-adsp_split/firmware/adsp*.* ${D}${nonarch_base_libdir}/firmware/
-    install -m 0444 ./21-cdsp_split/firmware/cdsp*.* ${D}${nonarch_base_libdir}/firmware/
+    install -m 0444 ./21-cdsp_split/firmware/cdsp*.* ${D}${nonarch_base_libdir}/firmware/qcom/db845c
```
Comment (ndechesne) [inline recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb]: ok, i am checking with bjorn if this is really the right place for this firmware.

File: recipes-support/adsprpc/adsprpc_git.bb (hunk 2 above)
Comment (Najon) [reply recipes-support/adsprpc/adsprpc_git.bb]: ok

File: recipes-support/adsprpc/adsprpc_git.bb~ (hunk 1 above)
Comment (Najon) [reply recipes-support/adsprpc/adsprpc_git.bb~]: thanks :)

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb (hunk 3 above)
Comment (Najon) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb]: The firmware path is given in the dts of the dragonboard here
https://git.linaro.org/landing-teams/working/qualcomm/kernel.git/tree/arch/arm64/boot/dts/qcom/sdm845-db845c.dts?h=release/db845c/qcomlt-5.2#n331

**Comment by Najon** [comment]
Also I don't know how qmic is required too.
Please tell me I have a patch which add it.
Thanks

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb (hunk 3 above)
Comment (ndechesne) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb]: yes, i know we have this path, but i am not sure if it's a temporary change or not. that's what i meant.

**Comment by ndechesne** [comment]
qmic is a native tool, we don't need it in the image.

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb (hunk 3 above)
Comment (ndechesne) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb]: I've updated the firmware-name property in the kernel, can you please update your PR accordingly? what we had decided was to put the ADSP and CDSP firmware in /lib/firmware/qcom/sdm845 (not db845c) and the kernel was wrong. I have also updated the kernel recipe.

Comment (Najon) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v2.bb]: Sure it's done.

Thanks for fixing the path in the dts.

**Comment by Najon** [comment]
just rebased the branch

**Comment by ndechesne** [comment]
thanks a lot!

**Pull request merged**